### ISSUE TITLE
🚚 Configure App Routing

### ISSUE THREAD

**Issue by IanOdhiambo9** [italanta/elewa-group#40]
#  Configure App Routing
---

#### Note: This issue is dependent on #1, please complete this issue first.

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user I want routes to navigate to various parts of the website

### Description

The website navigation is managed on apps/elewa-group-website/src/app

On the path above create a file named app.routing.ts

Paste the code below on that file 

```typescript

import { NgModule } from '@angular/core';
import { RouterModule, Route }  from '@angular/router';

export const ELEWA_GROUP_WEBSITE_ROUTES: Route[] = [

  // App Entry-Point

  { path: '', redirectTo: `home`, pathMatch: 'full' },
  {  path: 'home', redirectTo: 'home/en'  },

];

@NgModule({
  imports: [
    RouterModule.forRoot(
      ELEWA_GROUP_WEBSITE_ROUTES,
      {
        scrollPositionRestoration: 'enabled',
        enableTracing: true, 
        onSameUrlNavigation: 'reload'
      }
    )
  ],
  exports: [
    RouterModule
  ]
})
export class AppRoutingModule { }


```

Your assignment is to implement the routing for the various pages/modules of the website on the file above.

### Technical Analysis

This feature will be implemented on apps/elewa-group-website/src/app

### Acceptance Criteria

1. All the navigation routes are properly implemented and working.

**Comment by adamswonder** [comment]
Working on this issue with @JamesMGithire

**Comment by CharlesMuvaka** [comment]
Done the home and default route('')

**Comment by mbuimbogo** [comment]
Working on this with [Danstan ](https://github.com/Danstan-O)

**Comment by IanOdhiambo9** [comment]
This is on hold until further notice..
